public/javascripts/SVLabel/src/SVLabel/modal/ModalSkip.js

```diff
@@ -82,46 +68,41 @@ function ModalSkip(form, onboardingModel, ribbonMenu, taskContainer, tracker, ui
      * This method handles a click Redirect event.
      * @param e
      */
-     this._handleClickNewNeighborhood = function(e) {
+     this._handleClickNewNeighborhood = (e) => {
         tracker.push("ModalSkip_ClickRedirect");
          window.location.replace('/explore?newRegion=true&resumeRoute=false');
      };
 
-    /**
-     * This method handles a click Explore event.
-     * @param e
-     */
-     this._handleClickExplore = function(e) {
-        tracker.push("ModalSkip_ClickExplore");
-         uiModalSkip.firstBox.hide();
-         uiModalSkip.secondBox.show();
-     };
-
-    /**
-     * This method handles a click Cancel event on the first jump screen.
-     * @param e
-     */
-    this._handleClickCancelFirst = function(e) {
-        tracker.push("ModalSkip_ClickCancelFirst");
-        self.hideSkipMenu();
-    };
-
     /**
      * This method handles a click Cancel event on the second jump screen.
      * @param e
      */
-    this._handleClickCancelSecond = function(e) {
+    this._handleClickCancelSecond = (e) => {
```

Comment:
I think that `cancelSecond` should be renamed since there are no longer multiple different cancels to make!